x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/store/NativeRolesStore.java

```diff
@@ -456,7 +457,7 @@ static RoleDescriptor transformRole(String id, BytesReference sourceBytes, Logge
         try {
             // we pass true as last parameter because we do not want to reject permissions if the field permissions
             // are given in 2.x syntax
```

Comment:
This comment is outdated before the current change. But the current one made it slightly worse because now the last parameter is actually a boolean but it is not for 2.x format. Can we amend this comment?

Reply:
> Can we amend this comment?

Yes, I will adjust it.